src/ploomber_engine/tracking/tracking.py

```diff
@@ -175,7 +175,7 @@ def track_execution(
         raise click.ClickException(
             "Missing sklearn-evaluation: pip install sklearn-evaluation"
         )
-
+    parameters = parameters if type(parameters) is dict else dict()
```

Comment:
you can simplify this:

```suggestion
    parameters = parameters or dict()
```

This will work as we expect it:

```python

In [1]: None or {}
Out[1]: {}

In [2]: dict(a=1) or {}
Out[2]: {'a': 1}
```

Reply:
Ohh I thought if the user input parameter as `True`, so we just do the type check beforehand to ensure the parameters is valid dict()

But I think we can do `parameters = parameters or dict()` it it's rare case

Reply:
Fix in [105ea97](https://github.com/ploomber/ploomber-engine/pull/41/commits/105ea97f289ad928d44e5b2d18bb3d40e5e07b51)